app/assets/stylesheets/multi_page.scss

```diff
@@ -0,0 +1,42 @@
+$base        : #AAAAAA;
+$secondary-color: white;
+
+.flash {
+  background-color: maroon;
+  color: white;
+  font-size: 14pt;
+}
+
+.fullWidth {
+   width: 100%;
```

Comment:
Line should be indented 2 spaces, but was indented 3 spaces<br>Properties should be ordered margin-left, margin-right, max-width, width